write-fonts/src/write.rs

```diff
@@ -145,6 +145,16 @@ impl TableWriter {
         data.add_offset(obj_id, width, self.offset_adjustment);
     }
 
+    /// Add a padding byte of necessary to ensure the table length is an even number.
+    ///
+    /// This is necessary for things like the glyph table, which require offsets
+    /// to be aligned on 2-byte boundaries.
+    pub fn ensure_2byte_aligned(&mut self) {
```

Comment:
nit: I find ensure less informative than pad wrt what is actually going to happen, e.g. pad_to_2byte_aligned